data/src/test/java/org/apache/iceberg/data/DeletesReadTest.java

```diff
@@ -80,21 +95,37 @@ public void testEqualityDeletes() throws IOException {
     );
 
     DeleteFile eqDeletes = FileHelpers.writeDeleteFile(
-        table, Files.localOutput(temp.newFile()), Row.of(0), dataDeletes, deleteRowSchema);
+        testTable, Files.localOutput(temp.newFile()), Row.of(0), dataDeletes, deleteRowSchema);
 
-    table.newRowDelta()
+    testTable.newRowDelta()
         .addDeletes(eqDeletes)
         .commit();
 
     StructLikeSet expected = rowSetWithoutIds(29, 89, 122);
-    StructLikeSet actual = rowSet(table);
+    StructLikeSet actual = rowSet(testTable);
 
     Assert.assertEquals("Table should contain expected rows", expected, actual);
   }
 
+  protected void generateTestData() throws IOException {
```

Comment:
I think this method should no longer exist, and there is no reason for it to be accessible to subclasses. It is also not a good idea to write methods like this one that have assumptions about other variables and operate by creating side-effects. It is less of a problem now that it is not called by subclasses. Now, there is just little reason for it to exist when this could be moved into the `@Before` method.

Reply:
Okay, make sense to me. Moved to `@Before`.